src/server/controller/cycleData/persistNodeValue/calculateDependantNodes/calculateDependantNodes.ts

```diff
@@ -7,7 +7,10 @@ import { RowRepository } from '@server/repository/assessment/row'
 
 import { calculateNode } from './calculateNode'
 
-export const calculateDependantNodes = async (props: Props, client: BaseProtocol): Promise<NodeUpdates> => {
+export const calculateDependantNodes = async (
+  props: Omit<Props, 'value'>,
```

Comment:
Omit property `value` as it was not used and we did not have it in updateOriginalDataPoint